### ISSUE TITLE
router.project-osrm.org invalid certificate

### ISSUE THREAD

**Issue by moimael** [Project-OSRM/osrm-frontend-v2#147]
Hi,

When using https api for routing with Firefox OS, the request fails because the certificate is invalid.

Here is the error : 

```
router.project-osrm.org uses an invalid security certificate.

The certificate is not trusted because no issuer chain was provided.

(Error code: sec_error_unknown_issuer)
```

Using http, everything works fine.

Thanks

**Comment by perliedman** [comment]
Most likely a problem with the version of Firefox OS you're using. The certificate checks out fine in both Chrome and Firefox on both Windows and Linux.

**Comment by danpat** [comment]
@perliedman It looks like we're not serving up the intermediate certificates in the trust chain.  Most browsers come with them, but I guess Firefox OS only has the root certificate, and not the intermediaries.

We've got some feelers out to get the server updated.  Unfortunately, not a lot of people have access to it, and they're a bit tricky to get a hold of.

**Comment by perliedman** [comment]
@danpat alright, more than I knew about certs! Thanks.

**Comment by moimael** [comment]
Thank you both for your quick answers, indeed it seems to be an intermediate certificate problem !

**Comment by TheMarex** [comment]
Good to close here? :smile:

**Comment by danpat** [comment]
@TheMarex no, unless someone has fixed the certificate, then we're still serving up the wrong thing.
